pyuvdata/uvcal/uvcal.py

```diff
@@ -3676,13 +4512,21 @@ def __iadd__(
         run_check_acceptability : bool
             Option to check acceptable range of the values of parameters after
             combining objects.
+        ignore_name : bool
+            Option to ignore the name of the phase center (`cat_name` in
+            `phase_center_catalog`) when combining two UVData objects. If set to True,
+            phase centers that are the same up to their name will be combined with the
+            name set to the name found in the first UVData object in the sum. If set to
```

Comment:
UVData -> UVCal

Reply:
Fixed in 766e32646768181b70ddd8d2e576d90ca57c254c.